documentation/contribute/contributingDatabaseSupport.md

```diff
@@ -37,14 +37,20 @@ Flyway itself should start. Since Flyway doesn’t yet support your database you
 
 You’re now ready to start adding that database support. We’re going to assume your database platform is called **FooDb**. Change the obvious naming conventions to suit your database.
 
+## Project structure
+
+First of all, where should we put the changes? In versions of Flyway up to 7.10, we would make changes to the Flyway core. However, 
+we've more recently separated out database-specific code into separate modules, and in particular, community contributions should
+go into `flyway-community-db-support`. This means there won't be any changes affecting critical areas of the product code for 
+existing users.
+
 ## Let's code!
 
 Here are all the changes and additions you'll need to make:
 
-1.  Add an optional dependency to your JDBC driver to the top-level `.pom`
-1.  Add a dependency of your JDBC driver to the flyway-commandline `.pom` if you wish the driver to be automatically shipped with the Flyway product.
-1.  In `flyway.conf`, document the format of the JDBC connection url for your database. This is not necessary to make Flyway work but it will help adoption of your database!
-1.  Create a new folder `Foo` in `org.flywaydb.core.internal.database` to contain your new classes.
+1.  Add a dependency to your JDBC driver to the `flyway-community-db-support` `pom.xml`
+1.  Document the format of the JDBC connection url for your database. This is not necessary to make Flyway work but it will help adoption of your database!
+1.  Create a new folder `Foo` in `org.flywaydb.community.database.foo` to contain your new classes.
```

Comment:
Don't need `foo` in the package name (it may confuse people and end up in `foo.foo`